sdk/src/client.rs

```diff
@@ -7,26 +7,29 @@ use reqwest::{
     Client as HttpClient, Url,
 };
 use reqwest_middleware::ClientWithMiddleware as HttpClientWithMiddleware;
-use serde::{de::DeserializeOwned, Serialize};
+use rmp_serde::{Deserializer, Serializer};
+use serde::{Deserialize, Serialize};
 use sp1_core::stark::StarkGenericConfig;
 use twirp::Client as TwirpClient;
 
 use crate::{
-    proto::prover::{
-        CreateProofRequest, GetProofStatusRequest, GetProofStatusResponse, ProofStatus,
-        Sp1ProverServiceClient, SubmitProofRequest,
+    proto::network::{
+        CreateProofRequest, GetProofStatusRequest, GetProofStatusResponse, GetRelayStatusRequest,
+        GetRelayStatusResponse, NetworkServiceClient, ProofStatus, RelayProofRequest,
+        SubmitProofRequest, TransactionStatus,
     },
-    SP1ProofWithIO, SP1Stdin,
+    SP1ProofWithIO,
 };
 
-pub struct SP1ProverServiceClient {
+const DEFAULT_PROVER_NETWORK_RPC: &str = "https://rpc.succinct.xyz/";
+pub const SP1_VERIFIER_ADDRESS: &str = "0x9a39f368676f7a5cbbfe8ea33c258c4536b5398f";
```

Comment:
should this be configurable by env variable as well?

Reply:
updated so that you can override with SP1_VERIFIER_ADDRESS env var